index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+<h1 id= "title">Hello world!</h1> 
+<form id="survey-form">
+  <label type="text" id="name-label">Name</label>
+  <input type="text" id="name" placeholder="Name" required><br>
+  <label type="email" id="email-label">Email</label>
+  <input type ="email" id="email" placeholder="email" required><br>
+  <label type="number" id="number-label">Age</label>
+  <input type="number" id="number" min="18" max="75" placeholder="Age"><br>
+  <select id="dropdown">
```

Comment:
Nice coding here! I think you should try to wrap it with section tag "section" to make it look better and also to help with your project semantics.